docs/outlook/one-outlook.md

```diff
@@ -46,7 +46,7 @@ The following table identifies key Outlook scenarios and their support status in
-|Data loss prevention|Prevent users from forwarding mail items that contain highly sensitive information.|Partially supported. Essential features are yet to be addressed to create a similar experience to VSTO or COM add-ins.|[Automatically check for an attachment before a message is sent](smart-alerts-onmessagesend-walkthrough.md)<br><br>[Handle OnMessageSend and OnAppointmentSend events in your Outlook add-in with Smart Alerts](onmessagesend-onappointmentsend-events.md)<br><br>[Manage the sensitivity label of your message or appointment in compose mode](sensitivity-label.md)<br><br>[Verify the sensitivity label of a message](https://github.com/OfficeDev/Office-Add-in-samples/tree/main/Samples/outlook-verify-sensitivity-label)<br><br>[Office.SensitivityLabel](/javascript/api/outlook/office.sensitivitylabelscatalog)<br><br>[Office.SensitivitiyLabelsCatalog](/javascript/api/outlook/office.sensitivitylabelscatalog)<br><br>[Office.SensitivityLabelDetails](/javascript/api/outlook/office.sensitivitylabeldetails)|
+|Data loss prevention|Prevent users from forwarding mail items that contain highly sensitive information.|Partially supported. Essential features are yet to be addressed to create a similar experience to VSTO or COM add-ins.|[Automatically check for an attachment before a message is sent](smart-alerts-onmessagesend-walkthrough.md)<br><br>[Handle OnMessageSend and OnAppointmentSend events in your Outlook add-in with Smart Alerts](onmessagesend-onappointmentsend-events.md)<br><br>[Manage the sensitivity label of your message or appointment in compose mode](sensitivity-label.md)<br><br>[Verify the sensitivity label of a message](https://github.com/OfficeDev/Office-Add-in-samples/tree/main/Samples/outlook-verify-sensitivity-label)<br><br>[Office.SensitivityLabel](/javascript/api/outlook/office.sensitivitylabelscatalog)<br><br>[Office.SensitivitiyLabelsCatalog](/javascript/api/outlook/office.sensitivitylabelscatalog)<br><br>[Office.SensitivityLabelDetails](/javascript/api/outlook/office.sensitivitylabeldetails)<br><br>[Office.context.mailbox.item.inReplyTo](/javascript/api/outlook/office.messagecompose?view=outlook-js-preview&preserve-view=true#outlook-office-messagecompose-inreplyto-member)<br><br>[Office.context.mailbox.item.getConversationIndexAsync](/javascript/api/outlook/office.messagecompose?view=outlook-js-preview&preserve-view=true#outlook-office-messagecompose-getconversationindexasync-member(1))<br><br>[Office.context.mailbox.item.getItemClassAsync](/javascript/api/outlook/office.messagecompose?view=outlook-js-preview&preserve-view=true#outlook-office-messagecompose-getitemclassasync-member(1))|
```

Comment:
Note that you now have the option to use `ul` and `li` elements to list these items. Ditto for the other lists in this table.